openjdk.test.mauve/build.xml

```diff
@@ -359,18 +359,43 @@ limitations under the License.
 		<delete dir="${openjdk_test_mauve_src_dir}/gnu/testlet/org/omg/PortableServer/POAOperations"/>
 	</target>
 
-	<target name="get-source" depends="download-cvsclient, create-work-dir" unless="openjdk_test_mauve_already_built">
-		<exec dir="${openjdk_test_mauve_work_dir}" executable="java">
-		<arg value="-Dcvs.root=:pserver:anoncvs@sourceware.org:/cvs/mauve"/>
-		<arg value="-jar"/>
-		<arg value="${first_prereqs_root}/cvsclient/org-netbeans-lib-cvsclient.jar"/>
-		<arg value="export"/>
-		<arg value="-r"/>
-		<arg value="HEAD"/>
-		<arg value="mauve"/>
-</exec>
+	<condition property="isJavaHomeSet" value="true">
+	   <isset property="JAVA_HOME"/>
+    </condition>
+
+    <condition property="isTestJDKHomeSet" value="true">
+	  <isset property="TEST_JDK_HOME"/>
+    </condition>
+
+    <target name="get-source-using-java-home" if="isJavaHomeSet".........> 
```

Comment:
Please remove the "....." portion from the <target> definition. I just added it in my example to mean "any other attribute that may be needed for the target".

Reply:
Removed